app/CMakeLists.txt

```diff
@@ -196,23 +203,33 @@ if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_VecGeom)
       demo-loop/LDemoKernel.cu
     )
   endif()
-  add_library(celeritas_demo_loop
+  celeritas_add_library(celeritas_demo_loop
     demo-loop/LDemoIO.cc
     demo-loop/LDemoParams.cc
     demo-loop/LDemoRun.cc
     ${_cuda_src}
+    SHARED
```

Comment:
Perhaps put the `SHARED` into the `celeritas_add_library` since it's (for now) temporary?

Reply:
Actually now that we have `celeritas_add_library` it is simpler to add.